.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: swift
+osx_image: xcode9.1
+xcode_workspace: PlayerControls
+
+branches:
+ only:
+ - travis-integration
+
+before_install:
+- brew update
+- brew install sourcery
+- brew install carthage
+- cd Demo
+- carthage update --platform iOS
+- brew cask install fastlane
+- cd ..
```

Comment:
What is the reason of latest `cd ..`?

Reply:
Because the Cartfile is in Demo folder. And a fastlane folder is in a previous one. So if we will try to use command "fastlane verify" it would say that there is no Fastfile. It was decided to use this command to get back in the previous folder, instead of moving this folder into Demo. 
Should it be left this way or It is better to move fastlane folder into Demo?

Reply:
Than probably you should go to the folder before executing command.

Reply:
@RomanTysiachnik  - any updates?